src/uint/sqrt.rs

```diff
@@ -25,7 +25,7 @@ impl<const LIMBS: usize> UInt<LIMBS> {
             // Sometimes an increase is too far, especially with large
             // powers, and then takes a long time to walk back.  The upper
             // bound is based on bit size, so saturate on that.
-            let res = Limb::ct_cmp(Limb(xn.bits() as Word), Limb(max_bits as Word)) - 1;
+            let res = Limb::ct_cmp(Limb(xn.bits_vartime() as Word), Limb(max_bits as Word)) - 1;
```

Comment:
This use of `bits_vartime` seems a little concerning after the renaming? @tarcieri

Reply:
cc @mikelodder7

Reply:
Seems okay for now. Not a clear direction yet going forward